docs/cloud.md

```diff
@@ -45,34 +45,34 @@ Generate JWT token :
 JWT_TOKEN=$(ocm token)
 ```
 
-##### HTTP request
+### HTTP request
 
 ```bash
 JWT_TOKEN=$(curl https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token -d client_id=cloud-services -d grant_type=refresh_token -d refresh_token=${OFFLINE_TOKEN} | jq -r '.access_token')
```

Comment:
This references the offline token that is described in the `ocm-cli` section which feels like an alternative to this one. Maybe we should move the description of the offline token to the top level `User Authentication` section?

Reply:
yep